test/resources/test_collection_exercise.py

```diff
@@ -468,3 +474,30 @@ def test_single_collection_exercise_classifier_type_fail(self, mock_request):
         # Then
         self.assertEqual(response.status_code, 500)
         self.assertEqual(response_data['error']['status_code'], 500)
+
+    @requests_mock.mock()
```

Comment:
Missing test case, bad request when required field (user_description) is not given in the payload.